src/vs/editor/common/view/editorColorRegistry.ts

```diff
@@ -12,6 +12,10 @@ import { registerThemingParticipant } from 'vs/platform/theme/common/themeServic
  * Definition of the editor colors
  */
 export const editorLineHighlight = registerColor('editor.lineHighlightBackground', { dark: null, light: null, hc: null }, nls.localize('lineHighlight', 'Background color for the highlight of line at the cursor position.'));
+// {{SQL CARBON EDIT}}
```

Comment:
We have our own color registry file - would suggest just putting these there : 

https://github.com/microsoft/azuredatastudio/blob/main/src/sql/platform/theme/common/colorRegistry.ts#L7